src/StreamFactoryInterface.php

```diff
@@ -48,5 +16,5 @@ public function createStreamFromResource($body);
      *
      * @return StreamInterface
      */
-    public function createStreamFromString($body);
+    public function createStream($body = '');
```

Comment:
I wonder why there is no `attach` method for the Stream, when there is `detach` https://github.com/php-fig/http-message/blob/f6561bf28d520154e4b0ec72be95418abe6d9363/src/StreamInterface.php#L44 . It would have helped us :) .